tutorials/intro-to-geospatial-analysis/sample.js

```diff
@@ -0,0 +1,148 @@
+var length = chic.geometry().length();
+print (length)
+// Find permeter of feature
+var peri = chic.geometry().perimeter();
+print (peri)
+
+function performMap(feature) {
+ // Reduce number of vertices in geometry; the number is to specify maximum error in meters
+ var simple = feature.simplify(1000);
+ // Find centroid of geometry
+ var center = simple.centroid();
+ // Create buffer around geometry; the number represents the width of buffer in meters
+ var buff = center.buffer(500);
+ // return variable from function
+ return buff;
+}
+// Map function over Feature Collection
+var mappedCentroid = chicColl.map(performMap);
+Map.addLayer(mappedCentroid,{},"Mapped centroids")
+
+// Features
+// Create Geometry
+var varGeometry = ee.Geometry.Polygon(0, 0, 40, 30, 20, 20, 0, 0);
+// Create Feature from Geometry
+var varFeature = ee.Feature(varGeometry, {
+ Name: ['Feature name', 'Supreme'],
+ Size: [500, 1000]
+});
+// Get values of a property
+var arr=varFeature.get('Size');
+print (arr);
+// Select a subset of properties and rename them
+var varFeaturenew = varFeature.select(['Name'], ['Descriptor']);
+print(varFeaturenew);
+
+// Images
+Map.setCenter(-88, 41.8, 9);
+var raw = ee.ImageCollection('MODIS/006/MYD11A2');
+var roi = ee.FeatureCollection('users/tirthankar25/chicago');
+print(raw);
+// Select a band of the image collection using either indexing or band name
+var bandSel1 = raw.select(0);
+var bandSel2=raw.select('LST_Day_1km');
+// Filter the image collection by a date range
+var filtered=raw.filterDate('2002-12-30','2004-4-27');
+print(filtered);
+// Limit the image collection to the first 50 elements
+var limited = raw.limit(50);
+print(limited);
+print(bandSel1);
+// Calculate mean of all images (pixel-by-pixel) in the collection
+var mean = bandSel1.mean();
+// Calculate image of region of interest
+var clipped = mean.clip(roi);
+// mathematical operation on image pixels to convert from digital number of satellite observations to degree Celsius
+var calculate = clipped.multiply(.02).subtract(273.15);
+Map.addLayer(calculate,{min: 20, max: 30, palette: ['blue', 'green', 'red']},'LST');
+// Select pixel in the image that are greater than 30.8
+var mask = calculate.gt(30.8);
+Map.addLayer(mask,{},"mask");
+// Use selected pixels to mask the whole image
+var masked = clipped.mask(mask);
+Map.addLayer(masked,{min: 10, max: 30, palette: ['blue', 'green', 'red']},'LST_masked');
+
+// Image to table example
+// Function to find mean of pixels in region of interest
+var getRegions = function(image) {
+  // Load feature collection
+  var urban = ee.FeatureCollection('users/tirthankar25/chicago');
+ return image.reduceRegions({
+  // Collection to run operation over
+    collection: urban,
+  // Reducer to use
+  reducer: ee.Reducer.mean(),
+  // Pixel resolution used for the calculations
+  scale: 1000,
+ });
+};
+// Load image
+var image = ee.ImageCollection('MODIS/MYD13A1').filterDate('2002-07-08', '2017-07-08')
+ .mean().select('NDVI');
```

Comment:
Please put the select before the mean.  It's always good practice to select first.